frontend/data/roundData.js

```diff
@@ -14,16 +14,32 @@ const roundData = {
       sections: [
         {
           title: 'Time Commitment (7 points)',
-          description:
-            '-1 for each:\n' +
-            '- Exec member for another org Consulting Club such as IBC, OTCR\n' +
-            '- Some other club that requires time that I can’t think of(ex: Enactus Fraternity Pledge during the same semester)\n' +
-            '- Hard/Time-consuming classes\n' +
-            '\n' +
-            '-2 for each:\n' +
-            '- Hard course-load (still take one point off for each hard class they have as described above)\n' +
-            '- Too many org obligations\n',
-          prompt: 'Give them a score out of 7:',
+          description: [
+            React.createElement('b', {}, '-1 for each:'),
+            React.createElement('ul', {}, [
+              React.createElement(
+                'li',
+                {},
+                'Exec member for another org Consulting Club such as IBC, OTCR'
+              ),
+              React.createElement(
+                'li',
+                {},
+                'Some other club that requires time that I can&#39;t think of (ex: Enactus Fraternity Pledge during the same semester)'
+              ),
+              React.createElement('li', {}, 'Hard/Time-consuming classes')
+            ]),
+            React.createElement('b', {}, '-2 for each:'),
+            React.createElement('ul', {}, [
+              React.createElement(
+                'li',
+                {},
+                'Hard course-load (still take one point off for each hard class they have as described above)'
+              ),
+              React.createElement('li', {}, 'Too many org obligations')
+            ])
+          ],
```

Comment:
does jsx not work here?

Reply:
Aryn was trying this syntax earlier so I went with it - I don't think we could get other stuff working

Reply:
Yeah for some reason, when we worked with Aria and Roy, this worked while jsx was failing.